syntax = "proto3";

message Product{
    string item = 1;
    int32 num = 2;
}

message TransactionResponse{
    bool status = 1;
    int32 inventory = 2;
}

service ProductService{
    rpc CheckItem(Product) returns (TransactionResponse) {}
    rpc ChangeItem(Product) returns (TransactionResponse) {}
}